src/cms/static/js/pages/page_bulk_actions.js

```diff
@@ -31,3 +31,14 @@ function bulk_action_execute() {
         window.open(bulk_action_url+"?target_lang="+action+"&pages="+pages.join(','), "_blank");
     }
 }
+
+function short_url_to_clipboard(url) {
+    u('#mirrored_page_first_div').removeClass('hidden');
+    var copyText = document.getElementById("copy_to_clipboard");
+    copyText.value = "https://" + location.hostname + "/" + url;
+    u('#copy_to_clipboard').removeClass('hidden');
+    copyText.select();
+    copyText.setSelectionRange(0, 99999);
+    document.execCommand("copy");
+    u('#copy_to_clipboard').addClass('hidden');
+}
```

Comment:
This code currently exists twice. Have not yet found a place where to put it in order to merge it.

Reply:
I started to separate the js files by tasks in `src/cms/static/js/pages/`, so you could just create a new file `src/cms/static/js/pages/copy_to_clipboard.js` and include this in the page tree template...

Reply:
I modified the function to be used in other cases as well and moved it into a dedicted JS file.